### PR TITLE
added a console payment tnx object

### PR DESCRIPTION
**Pull request by Mlg29**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I need help writing tests.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by uchibeke** [comment]
@Mlg29 do you mind resolving the conflict?

**Comment by Mlg29** [comment]
@uchibeke okay. I will look into it

File: packages/offline-pay-with-crypto-widget/src/components/Modal/index.js [hunk 1]
```diff
@@ -36,30 +45,79 @@ const dropIn = {
 
 
 export function Modal({ config }) {
```
Comment (uchibeke) [inline packages/offline-pay-with-crypto-widget/src/components/Modal/index.js:47]: Please see new config https://github.com/Chimoney/offline-pay-with-celo/blob/main/packages/offline-pay-with-crypto-widget/src/components/Modal/index.js#L39

File: packages/offline-pay-with-crypto-widget/src/components/Modal/index.js [hunk 2]
```diff
@@ -36,30 +45,79 @@ const dropIn = {
...
 
-  // eslint-disable-next-line no-unused-vars
-  const { name, order_type, amount, currency, address, store_img } = config;
+  const [selectedCoin, setSelectedCoin] = useState(null)
+
+const paymentInfo = async () => {
+        let goldtoken = await kit.contracts.getGoldToken()
+        const oneGold = kit.web3.utils.toWei('1', 'ether')
+
+    let anAddress = '0x998f2193ae2d887249c27d2ab2ceb9e60be75f7b'
```
Comment (uchibeke) [inline packages/offline-pay-with-crypto-widget/src/components/Modal/index.js:63]: We do not want to hardcode. New changes fixes this https://github.com/Chimoney/offline-pay-with-celo/blob/main/packages/offline-pay-with-crypto-widget/src/components/Modal/index.js#L39

**Review by uchibeke — CHANGES_REQUESTED** [review]
Could you review the latest main branch. We might not need this change again or try to merge your web3 stuff with what we have in main, if possible

**Pull request closed without merging**